internal/p2p/p2ptest/require.go

```diff
@@ -136,6 +136,7 @@ func RequireUpdate(t *testing.T, peerUpdates *p2p.PeerUpdates, expect p2p.PeerUp
 
 	select {
 	case update := <-peerUpdates.Updates():
+		update.Channels = nil
```

Comment:
This I presume is just so we compare correctly independent of the contents of this field?

Reply:
yeah, I've refactored this so that we just compare the node id and the status.